### PR TITLE
adding estimated reading time to email

### PR DESCRIPTION
**Pull request by martinraison**
@yingjieMiao it's not easy to test email templates outside prod for the moment, so I'd like to push it now to see if anything is wrong. Let me know if you have any remarks about the code

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala [hunk 1]
```diff
@@ -22,6 +23,9 @@ class WordCountCommanderImpl @Inject()(
   wordCountCache: NormalizedURIWordCountCache
 ) extends WordCountCommander{
 
+  private val WORDS_PER_MINUTE = 250
+  private val READ_TIMES = Seq(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 30, 60)
+
```
Comment (yingjieMiao) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala:28]: nice!

**Comment by eishay** [comment]
wohoooo!!!

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala [hunk 2]
```diff
@@ -56,4 +60,11 @@ class WordCountCommanderImpl @Inject()(
       case None => getFromScraper(id, url)
     }
   }
+
+  def getReadTimeMinutes(id: Id[NormalizedURI], url: String): Future[Int] = {
```
Comment (eishay) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala:64]: do you have a test case for that? 
Cool implementation :-)

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala [hunk 3]
```diff
@@ -56,4 +60,11 @@ class WordCountCommanderImpl @Inject()(
...
+    getWordCount(id, url) map { estimate =>
```
Comment (yingjieMiao) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala:65]: note that estimate could be -1 (scrape failed, etc). we want to hide this when count <=0 . 

maybe we can have a wrapper class.

Comment (martinraison) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/WordCountCommander.scala:65]: just saw this comment - I have a pushed fix already, actually

**Pull request merged**